databroker/experimental/server_ext.py

```diff
@@ -121,6 +121,17 @@ def put_data(self, body):
         if result.matched_count != result.modified_count:
             raise ValueError("Error while writing to database")
 
+    def delete(self):
+        if self.doc.data_url is not None:
+            path = self.doc.data_url.path
+            if platform == "win32" and path[0] == "/":
+                path = path[1:]
+
+            self.file.close()
+            os.remove(path)
```

Comment:
I think `path.unlink()` would be a safer (future-proof) choice here.

Reply:
Fixed in https://github.com/bluesky/databroker/pull/715/files#diff-d4869bded9d2c76766f74b02b105f0f5e2825963da33e28a1a566e5a8af6f6b0R131